main/ble_server.c

```diff
@@ -38,24 +39,24 @@
 #define SPP_PROFILE_NUM             1
 #define SPP_PROFILE_APP_IDX         0
 #define ESP_SPP_APP_ID              0x56
-#define DEVICE_NAME_GAP          "BLE_TO_ISOTP"    //The Device Name Characteristics in GAP
-#define SPP_SVC_INST_ID	            0
+#define DEVICE_NAME_GAP             "BLE_TO_ISOTP"    //The Device Name Characteristics in GAP
+#define SPP_SVC_INST_ID             0
 
 /// SPP Service
-static const uint16_t spp_service_uuid = 0xABF0;
+static const uint16_t spp_service_uuid = 0xABF0; // '0000abf0-0000-1000-8000-00805f9b34fb'
 /// Characteristic UUID
-#define ESP_GATT_UUID_SPP_DATA_RECEIVE      0xABF1
-#define ESP_GATT_UUID_SPP_DATA_NOTIFY       0xABF2
-#define ESP_GATT_UUID_SPP_COMMAND_RECEIVE   0xABF3
-#define ESP_GATT_UUID_SPP_COMMAND_NOTIFY    0xABF4
+#define ESP_GATT_UUID_SPP_DATA_RECEIVE      0xABF1 // '0000abf1-0000-1000-8000-00805f9b34fb'
+#define ESP_GATT_UUID_SPP_DATA_NOTIFY       0xABF2 // '0000abf2-0000-1000-8000-00805f9b34fb'
+#define ESP_GATT_UUID_SPP_COMMAND_RECEIVE   0xABF3 // '0000abf3-0000-1000-8000-00805f9b34fb'
+#define ESP_GATT_UUID_SPP_COMMAND_NOTIFY    0xABF4 // '0000abf4-0000-1000-8000-00805f9b34fb'
 
 static const uint8_t spp_adv_data[23] = {
     /* Flags */
     0x02,0x01,0x06,
     /* Complete List of 16-bit Service Class UUIDs */
     0x03,0x03,0xF0,0xAB,
     /* Complete Local Name in advertising */
-    0x0F,0x09, 'B', 'L', 'E', '_', 'T', 'O', '_', 'I', 'S', 'O', 'T','P', '2', '0'
```

Comment:
i think this was a bug/it didn't match before. needed to for sure match `DEVICE_NAME_GAP` i think